src/views/HomePage.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import BG from '../../media/images/BgAeivsio.webp';
+import { useForm } from 'react-hook-form';
+import { Link } from 'react-router-dom';
+
+function HomePage(): JSX.Element {
+  const { register, handleSubmit } = useForm();
+
+  return (
+    <div
+      className="h-screen w-screen p-28"
+      style={{
+        backgroundImage: `url(${BG})`,
```

Comment:
The wallpaper does not take the full width on a 24 inch screen

![Capture d’écran 2021-06-18 à 09 59 25](https://user-images.githubusercontent.com/79398461/122527734-e505f780-d01b-11eb-8781-c900c7072afe.png)